CHANGELOG.md

```diff
@@ -0,0 +1,43 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- `CHANGELOG.md` documenting notable changes to this project
```

Comment:
should we keep track of the ticket number on each of the entries, or do we not care?

Reply:
I have no opinion on that. I can see arguments for both. Let's first go without and alter if we see necessary?